### PR TITLE
[MMCli] Add Metafile

### PR DESCRIPTION
**Pull request by kennymckormick**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,3 @@
+include mmcls/model_zoo.yml
```
Comment (innerlee) [inline MANIFEST.in]: sorry this is not `mmcls` repo

Comment (kennymckormick) [reply MANIFEST.in]: already corrected

**Comment by innerlee** [comment]
Ping me when it is ready to merge 😃

**Comment by kennymckormick** [comment]
This PR is ready and tested. Currently, we only add some essential fields in the metafile. We will expand the field set in the future. @innerlee

**Review by innerlee — APPROVED** [review]

**Pull request merged**